crates/services/gas_price_service/src/fuel_gas_price_updater/fuel_core_storage_adapter.rs

```diff
@@ -51,41 +34,6 @@ mod l2_source_tests;
 
 pub mod storage;
 
-impl<Storage> MetadataStorage for StructuredStorage<Storage>
```

Comment:
The main idea of this code was that gas price crate defined default implementation of the `MetadataStorage` port. I don't see why we don't want to follow that idea=)

Reply:
Having default implementation allows us test it within this crate, and inherit it later in the `fuel-core`, if we want to, of course=)

Reply:
retained in [100b5b8](https://github.com/FuelLabs/fuel-core/pull/2226/commits/100b5b86e092e00e466befb1070d570fa0935f8e)